### ISSUE TITLE
DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead

### ISSUE THREAD

**Issue by pvsa** [egeland/nagios-rbl-check#26]
When running  nagios-rbl-check # ./check_rbl.py -h mx4 -w 0 -c 1
I get message:
...nagios-rbl-check/./check_rbl.py:271: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
  t.setDaemon(True)

Repo is up to date

**Comment by smashedr** [comment]
The fix is simple.

```diff
     # Spawn a pool of threads then pass them the queue
     for i in range(10):
         t = ThreadRBL(queue)
-        t.setDaemon(True)
+        t.daemon = True
         t.start()
```

Just a warning as of 3.11. I will get a PR in and see if this passes all tests when I get a chance.

**Comment by pvsa** [comment]
Hi,Thanks for the hint. Short Test with python3.10, python3.11 works like a Charme.I inform you if some failures come up.BrPAm 19.06.2023 um 22:36 schrieb SmAsHeD ***@***.***>:﻿
The fix is simple.
     # Spawn a pool of threads then pass them the queue
     for i in range(10):
         t = ThreadRBL(queue)
-        t.setDaemon(True)
+        t.daemon = True
         t.start()
Just a warning as of 3.11. I will get a PR in and see if this passes all tests when I get a chance.

**Comment by egeland** [comment]
Closed in #27 - nice work, folks!
I have to say, it's really cool that this little thing I created so long ago now is still being used! 😀
